### PR TITLE
Feature/regression plot

### PR DESCRIPTION
**Pull request by tdesfont**
Add the javascript code in app.js to have a view of the prediction distribution

### CODE REVIEW

File: webapps/model-drift-view/app.js [hunk 1]
```diff
@@ -344,53 +344,7 @@ function draw_KDE_regression(data) {
...
-                .y(d => y(d[1]))
-            );
-    }
-
-    // Listen to the slider?
-    d3.select("#label-list").on("change", function(d) {
-        selectedGroup = this.value
-        updateChart(selectedGroup)
-    });
+        .text("Density probability function of the prediction");
```
Comment (du-phan) [inline webapps/model-drift-view/app.js]: `Probability density function`

File: python-lib/dku_data_drift/drift_analyzer.py [hunk 2]
```diff
@@ -9,13 +9,12 @@
...
 from dku_data_drift.model_tools import format_proba_density
 
-
 logger = logging.getLogger(__name__)
 
 ORIGIN_COLUMN = '__dku_row_origin__'  # name for the column that will contain the information from where the row is from (original test dataset or new dataframe)
 FROM_ORIGINAL = 'original'
 FROM_NEW = 'new'
-MIN_NUM_ROWS = 500 # heuristic choice
+MIN_NUM_ROWS = 10 # heuristic choice
```
Comment (du-phan) [inline python-lib/dku_data_drift/drift_analyzer.py:17]: Why did you change this to 10 ?

File: webapps/model-drift-view/app.js [hunk 3]
```diff
@@ -64,9 +64,32 @@ function markRunning(running) {
...
+            } else {
+                d3.select("#feature_importance_div").style('display', 'none');
+            }
+            break;
+        case "REGRESSION":
+            d3.select("#fugacity_div").selectAll("div").remove();
+            d3.select("#kde_container_div").select("h3").remove();
+            draw_KDE_regression(data['kde']);
+            drawFeatureImportance(data['feature_importance']);
+            if (data.drift_accuracy > 0.1){
```
Comment (du-phan) [inline webapps/model-drift-view/app.js:83]: Let's do it >= 0.1 so that it fits with the condition in the beginning (>=0.1 -> medium drift)

File: custom-recipes/compare-datasets/recipe.json [hunk 4]
```diff
@@ -109,6 +109,12 @@
             ],
             "defaultValue": "single_column",
             "visibilityCondition": "model.prediction_column_available"
+        },
+        {
+            "name": "columns_to_remove",
+            "label": "Remove those columns from drift analysis",
```
Comment (du-phan) [inline custom-recipes/compare-datasets/recipe.json:115]: `Columns to ignore` sounds better and more compact

File: python-lib/dku_data_drift/drift_analyzer.py (hunk 2 above)
Comment (tdesfont) [reply python-lib/dku_data_drift/drift_analyzer.py:17]: It's a mistake thank you for spotting
